scripts/terraform/user-data.tpl

```diff
@@ -20,6 +20,8 @@ mount /var/lib/gravity/planet/etcd
 chown -R 1000:1000 /var/lib/gravity /var/lib/gravity/planet/etcd
 
 # Explicitly configure required parameters
+modprobe iptable_nat || true
+modprobe iptable_filter || true
```

Comment:
I think you forgot to add these to the list of modules to load on startup (see below)